### DISCUSSION TITLE
Упростить правила Firewall для iptables

### DISCUSSION THREAD

**Discussion by jorikfon** [mikopbx/Core#135] [Ideas (Идеи)]
Я вот думал что iptables правила можно было бы упростить визуально, если разрешено для 0.0.0.0/0 то для всех остальных подсетей уже не создавать разрешения для этого же протокола и порта. Сразу список iptables -S станет читаемым с одной страницы. А вот если для 0.0.0.0 не разрешен тогда уже по подсетям расписывапть. Плюс куча дублирующих и непонятных, например зачем у нас для 127.0.0.0 прописаны правила, пусть будет все разрешено одной строчкой.

Дописать Core/src/Core/System/Configs/IptablesConf.php
 
да отсортировать бы или по порту или по подсети 
а то в перемешку наверное надо получить все разрешённые, свернуть таблицу, отсортировать, а потом уже в iptables пихать

<img width="703" alt="NBScreenShot 2020-09-22 at 13 32 27" src="https://user-images.githubusercontent.com/5922918/126474241-9dbd8fc4-0524-47c4-8144-4d098c5ea891.png">

**Comment by sdocent** [comment]
Список правил можно еще сократить. Для каждой подсети указывать только две строки с разными протоколами, а порты через запятую
Паример; 
-A INPUT -s 172.16.34.0/24 -p tcp -m tcp --dport 5060,80,443,22,5038,8088,8089 -j ACCEPT
-A INPUT -s 172.16.34.0/24 -p udp -m udp --dport 5060,10000-10500, -j ACCEPT

**Comment by boffart** [comment]
создана задача https://github.com/mikopbx/Core/issues/236
